text/0118-search-api.md

```diff
@@ -275,16 +275,16 @@ This parameter is applied to the fields from `attributesToHighlight`. If there a
 
 Defines document attributes to be cropped. Cropped attributes have their values shortened around query terms.
 
+If `attributesToCrop` is present in the search query, the search results will include a `_formatted` object containing the attributes and their cropped parts. For more detailed regarding the `_formatted` behavior, see the [3.2.1.1.2. `_formatted`](#32112-formatted) section.
+
+If `"*"` is provided as a value (`attributesToCrop=["*"]`), all the attributes present in `displayedAttributes` setting will be cropped.
+
```

Comment:
I followed the same "format" as for `attributesToHighlight`. First, talking and redirect to the formatted section, then the `*` value behavior, and finally the available parameters.